nb-api/src/main/java/io/nosqlbench/api/engine/util/SSLKsFactory.java

```diff
@@ -81,18 +84,25 @@ public SSLContext getContext(NBConfiguration cfg) {
         if (sslParam.isPresent()) {
             String tlsVersion = cfg.getOptional("tlsversion").orElse(DEFAULT_TLSVERSION);
 
-            KeyStore keyStore;
+            KeyStore keyStore = null;
             char[] keyPassword = null;
-            KeyStore trustStore;
+            KeyStore trustStore = null;
+
+            boolean activeSSLValidation = true;
+            if (cfg.getOptional("sslValidation").isPresent() && cfg.getOptional("sslValidation")
+                    .get().equals("false")) {
+                logger.trace("sslValidation=false, skipping SSL validation.");
+                activeSSLValidation = false;
+            }
 
-            if (sslParam.get().equals("jdk")) {
+            if (activeSSLValidation && sslParam.get().equals("jdk")) {
```

Comment:
It's not clear to me how this is achieving the same effect as suggested by the initial issue. It looks like if you don't enable validation that we completely disregard trust store init. Since changes here are difficult to test in myriad configurations, I want to be extra careful with this are of the code.